src/components/DataTable/index.js

```diff
@@ -30,16 +30,23 @@ export function DataTable ({
 	customButton, customComponent,
 	customRowStyle, customRowClassName,
 	customCardBodyComponent,
-	limitValues = [10, 15, 25, 50],
-	contentLoader = true, category
-	}) {
+	limitValues = [5, 10, 15, 20, 50],
```

Comment:
@Aringoaway What is the default? I mean when there is no height passed to the index.js, there should be some default,, right? I propose `10`